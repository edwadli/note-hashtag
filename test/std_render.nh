tr = init track { durations = [ q q q q q q w ]; chords = [ 1 2 3 4 5 (init chord) (1,3,5) ] }
so = init song { tracks = { { tr } }; volumes = { 1.0 } }
Render so "test-render.wav"
PrintEndline "I didn't crash and now you have a song!"
