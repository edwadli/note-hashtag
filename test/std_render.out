I didn't crash and now you have a song!
